sofl/importer/source.py

```
@@ -132,3 +133,20 @@ def executable_format(self) -> str:
         raise NotImplementedError(
             f"No URL handler command available for {sys.platform}"
         )
+
+
+class OnlineFixSource(Source):
+    """Source for Online-Fix games"""
+    
+    source_id = "online-fix"
+    name = _("Online-Fix")
```

Ruff (0.8.2)

Undefined name `_`

(F821)